etl/steps/data/garden/papers/2023-04-21/broadberry_et_al_2015.meta.yml

```diff
@@ -0,0 +1,6 @@
+dataset:
+  {}
+
+tables:
+  {}
+
```

Comment:
I think you are not using this file. Since it has not information, we can delete it.

Reply:
Done!